packages/react-native/ReactAndroid/src/main/java/com/facebook/react/devsupport/LogBoxModule.kt

```diff
@@ -23,20 +23,13 @@ public class LogBoxModule(
       devSupportManager.createSurfaceDelegate(NAME)
           ?: LogBoxDialogSurfaceDelegate(devSupportManager)
 
-  /**
-   * LogBoxModule can be rendered in different surface. By default, it will use LogBoxDialog to wrap
-   * the content of logs. In other platform (for example VR), a surfaceDelegate can be provided so
-   * that the content can be wrapped in custom surface.
-   */
```

Comment:
Can we not lose this comment?

Reply:
Of course, I've committed the changes to restore the deleted comment. Please review the comment to ensure it's correct.